lib/compress/zstd_compress_internal.h

```diff
@@ -895,15 +895,55 @@ MEM_STATIC ZSTD_dictMode_e ZSTD_matchState_dictMode(const ZSTD_matchState_t *ms)
             ZSTD_noDict;
 }
 
+/* Defining this macro to non-zero tells zstd to run the overflow correction
+ * code much more frequently. This is very inefficient, and should only be
+ * used for tests and fuzzers.
+ */
+#ifndef ZSTD_WINDOW_OVERFLOW_CORRECT_FREQUENTLY
```

Comment:
Should this macro be documented ?
I presume it's only meant to be used by maintainers.

Reply:
I don't think it really deserves documentation in any `README.md`, since it shouldn't be enabled in anything but tests. It is automatically enabled for fuzzers, which have a standard macro `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`.

If we wanted this to be used more widely, I think we would want to have a general debug flag like `DEBUGLOG`, except for enabling "expensive" checks.